src/main/java/org/terasology/Sample/blocks/RomanColumnFamily.java

```diff
@@ -50,15 +88,112 @@ private Block getProperBlock(WorldProvider worldProvider, Vector3i location) {
             }
         }
         if (sides.contains(Side.BOTTOM) && !sides.contains(Side.TOP)) {
-            return blocks.get(SideBitFlag.getSide(Side.TOP)); // Returns a block that should be on top of the stack
+            return BLOCKS.get(SideBitFlag.getSide(Side.TOP)); // Returns a block that should be on top of the stack
         } else if (!sides.contains(Side.BOTTOM) && sides.contains(Side.TOP)) {
-            return blocks.get(SideBitFlag.getSide(Side.BOTTOM)); // Returns a block that should be on the bottom of the stack
+            return BLOCKS.get(SideBitFlag.getSide(Side.BOTTOM)); // Returns a block that should be on the bottom of the stack
         } else if (sides.contains(Side.BOTTOM) && sides.contains(Side.TOP)) {
-            return blocks.get(SideBitFlag.getSides(Side.BOTTOM, Side.TOP)); // Returns a normal column block
+            return BLOCKS.get(SideBitFlag.getSides(Side.BOTTOM, Side.TOP)); // Returns a normal column block
         } else if (!sides.contains(Side.BOTTOM) && !sides.contains(Side.TOP)) {
-            return blocks.get(SideBitFlag.getSides(Side.BOTTOM, Side.TOP)); // Returns a normal column block
+            return BLOCKS.get(SideBitFlag.getSides(Side.BOTTOM, Side.TOP)); // Returns a normal column block
         }
         return null; // If we return null, something has gone terribly wrong
     }
 
+    private Block makeABlock(BlockFamilyDefinition definition, BlockBuilderHelper blockBuilder, String section, BlockUri blockUri, byte sides) {
```

Comment:
maybe use a better name like addBlock() and use this function to add to your map.